src/main/java/seedu/address/model/appointment/FilterAppointmentPredicate.java

```diff
@@ -19,12 +19,18 @@ public FilterAppointmentPredicate(StartDateTime lowerBoundDateTime, EndDateTime
         this.upperBoundDateTime = upperBoundDateTime;
     }
 
+    private boolean isAppointmentFullyBeforeLowerBound(Appointment appointment) {
+        return appointment.getEndDateTime().isBefore(lowerBoundDateTime);
+    }
+
+    private boolean isAppointmentFullyAfterUpperBound(Appointment appointment) {
+        return appointment.getStartDateTime().isAfter(upperBoundDateTime);
+    }
```

Comment:
sorry to nitpick, but why did you define separate functions for this and not just define variables within the test function?

Reply:
> sorry to nitpick, but why did you define separate functions for this and not just define variables within the test function?

oh i think if it is private then there should be no problem